devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -58,7 +59,7 @@ metadata {
 }
 
 def initialize() {
-	if (isAeotec() || isNeoCoolcam() || isDome()) {
+	if (isAeotec() || isNeoCoolcam() || isDome() || isLeakGopher()) {
 		// 8 hour (+ 2 minutes) ping for Aeotec, NEO Coolcam, Dome
```

Comment:
aaaand now also for LeakGopher 😺 (same for below else...if statement change)

Reply:
Done :)